test/fetch/examples/get-shipments.example.ts

```diff
@@ -0,0 +1,44 @@
+// noinspection JSUnusedGlobalSymbols
```

Comment:
Is dit gebruikelijk?

Reply:
Ja dit word oa door de auto mock implementation gemaakt.